src/pages/index.js

```diff
@@ -183,18 +185,11 @@ function Home() {
                   "button button--outline button--secondary button--lg button--hero get-started-use-case get-started-use-case-2",
                   styles.getStarted
                 )}
-                to={useBaseUrl("docs/guides/orchestrate-human-tasks/")}
-              >
-                Human Task Orchestration
-              </Link>
-              <Link
-                className={clsx(
-                  "button button--outline button--secondary button--lg button--hero get-started-use-case get-started-use-case-2",
-                  styles.getStarted
+                to={useBaseUrl(
+                  "docs/guides/getting-started-orchestrate-human-tasks/"
                 )}
-                to={useBaseUrl("docs/guides/orchestrate-api-endpoints/")}
               >
```

Comment:
But one of the buttons didn't have a 1.3 counterpart, so I removed that button completely.